frontend/packages/console-plugin-sdk/src/typings/pages.ts

```diff
@@ -63,6 +63,8 @@ namespace ExtensionProperties {
     path: string | string[];
     /** Perspective id to which this page belongs to. */
     perspective?: string;
+    /** rootWindow indicate rendering as root window instead of layout content. */
+    rootWindow?: boolean;
```

Comment:
I think `rootWindow` is a bit ambiguous. Maybe `noLayout` ?

Reply:
Honestly i don't think a concise variable name will be descriptive enough.
IMO I'd suggest `root?: boolean` as `rootWindow` doesn't provide any more description than the other.
OR to make it even more useful for other uses, we could do `context?: string` and let the consumer of the extension supply the context.